talk/owt/sdk/p2p/objc/P2PPeerConnectionChannelObserverObjcImpl.h

```diff
@@ -22,6 +22,7 @@ class P2PPeerConnectionChannelObserverObjcImpl
   void OnStreamAdded(
       std::shared_ptr<owt::base::RemoteStream> stream) override;
   void OnStopped(const std::string& remote_id) override;
+  void OnPeerConnectionClosed(const std::string& remote_id) override;
```

Comment:
Other places where we use this method are protected with `#ifdef OWT_CG_SERVER`, like here:
https://github.com/open-webrtc-toolkit/owt-client-native/blob/adad03d829d7810b0093fb17f24afe43744d1fab/talk/owt/sdk/p2p/p2pclient.cc#L456
Should this protection be added here as well?

Reply:
https://github.com/open-webrtc-toolkit/owt-client-native/blob/adad03d829d7810b0093fb17f24afe43744d1fab/talk/owt/sdk/include/cpp/owt/p2p/p2pclient.h#L67

Quoting @taste1981 offline reply: "the code line you're listing is API for P2PClientOsbserver, not the same API jianjun is handling. There're multiple API with the same name on different classes."

Reply:
There are two places named `OnPeerConnectionClosed`, one is public (in `P2PClientObserver`). It's protected by `OWT_CG_SERVER`. Another one is not public, defined in `P2PPeerConnectionChannelObserver`. `P2PPeerConnectionChannelObserverObjcImpl` implements the latter one.